R/add_stat.R

```diff
@@ -148,6 +148,8 @@ add_stat <- function(x, fns, location = everything() ~ "label") {
     dplyr::tibble(variable = names(fns)) |>
     dplyr::mutate(
       summary_type = map_chr(.data$variable, ~ x$inputs$type[[.x]]),
```

Comment:
@ddsjoberg this was causing a break in even the original unit tests for `add_stat()` for `tbl_continuous()` objects, so I made a modification to `tbl_continuous.R` to at least get it "back to working" as it was previously